homeassistant/components/github/sensor.py

```diff
@@ -174,6 +181,7 @@ def __init__(self, repository, access_token=None, server_url=None):
         self.available = False
         self.latest_commit_message = None
         self.latest_commit_sha = None
+        self.latest_release_tag = None
```

Comment:
This isn't used.

Reply:
For my education: why not? It is used for all other attributes?

Reply:
This class is the data class not the entity class. We're not changing the data class.

Reply:
ok thanks. done